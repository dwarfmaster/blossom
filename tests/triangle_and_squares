12 17
0 8
8 9
9 4
4 0
0 1
1 7
7 0
1 10
10 11
11 2
2 1
3 6
6 2
2 3
5 3
3 4
4 5
