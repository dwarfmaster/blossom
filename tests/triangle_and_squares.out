0 -- 8
9 -- 4
1 -- 7
10 -- 11
6 -- 2
5 -- 3
